### DISCUSSION TITLE
Add Anghami music service support

### DISCUSSION THREAD

**Discussion by MWUIA** [music-assistant/support#2804] [Feature requests and ideas]
Anghami is a music streaming provider in the arab world. Can this be integrated as another music provider?

**Comment by MWUIA** [comment]
Any update on this?

**Reply by OzGav** [reply]
No. With only three votes this is low on the priority list. You would need someone to pick this up.

**Reply by OzGav** [reply]
Further update. I contacted them asking for an API key and received no reply. Those that are keen for this should try and contact them and suggest they support us to provide access for their users.

**Reply by MWUIA** [reply]
Ok, thx
